doc/tutorial.rst

```diff
@@ -201,3 +201,24 @@ There are two common cases. The span of the wing is in, y direction (rotation ab
   * positive rotation (+deltaAlpha) will pitch the wing downwards
 
 
+Kill Signals
+------------
+ADflow has two defined kill signals that can stop or kill ADflow gracefully without the loss of data. The two signals defined are
+
+  * ``-USR1`` - instructs ADflow to write a solution file after the current iteration
+  * ``-USR2`` - instructs ADflow to write a solution file and the computation will be stopped.
+
+The definition of an iteration is different for steady and unsteady. For steady it means after the current iteration, for unsteady after the current time step.
+
+The signals are enabled by default but can be switched off or disabled at compile time using the compiler flag ``-DUSE_NO_SIGNALS``.
+
+To use the signals from the command line run::
+
+  kill -USR1 <PID>
+
+where ``<PID>`` is the process id of the mpi process. These signals are often used when debugging. For instance, the ``-USR1`` signal can be useful to write out a semi-converged solution for further investigation, and the ``-USR2`` can be used to stop a stalled solution without loss of data. Other use-cases are also possible.
```

Comment:
Could you clarify how to find the `<PID>` on both our local machines and HPC?

Reply:
I assume you can use `top` on a compute node the same as locally?

Reply:
Yes, one could use `top`, or `ps -ef` for example. Added that to the docs.

Reply:
Yeah I mentioned this in the HPC guide -- you can just ssh into a compute node that's running your job, and monitor/send signals as you wish.